src/app/routes/article/getInitialData/index.test.js

```diff
@@ -1,5 +1,5 @@
-import getInitialData from './getInitialData';
-import articleJson from '../../../../data/pidgin/articles/cwl08rd38l6o.json';
+import getInitialData from '.';
+import articleJson from '../../../../../data/pidgin/articles/cwl08rd38l6o.json';
```

Comment:
We have a webpack alias for this directory - `#data`
```suggestion
import articleJson from '#data/pidgin/articles/cwl08rd38l6o.json';
```